targets/TARGET_ARM_SSG/TARGET_MUSCA_S1/PinNames.h

```diff
@@ -0,0 +1,117 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2020 Arm Limited
+ *
```

Comment:
Can you add SPDX identifiers to new files?

Reply:
Don't we have CI to catch SPDX missing? What happened with CI?

Reply:
Travis runs it but not if the tools fail. Looking at the order, we might want to check frozen tools as the last check, I'll create a PR now

Reply:
```
Found files with missing license details, please review and fix
File: /targets/TARGET_ARM_SSG/TARGET_MUSCA_S1/i2c_api.c reason: Missing SPDX license identifier
File: /targets/TARGET_ARM_SSG/TARGET_MUSCA_S1/lp_ticker.c reason: Missing SPDX license identifier
File: /targets/TARGET_ARM_SSG/TARGET_MUSCA_S1/mbed_serial_platform.c reason: Missing SPDX license identifier
File: /targets/TARGET_ARM_SSG/TARGET_MUSCA_S1/pinmap.c reason: Missing SPDX license identifier
File: /targets/TARGET_ARM_SSG/TARGET_MUSCA_S1/serial_api.c reason: Missing SPDX license identifier
File: /targets/TARGET_ARM_SSG/TARGET_MUSCA_S1/device/cmsis.h reason: Missing SPDX license identifier
File: /targets/TARGET_ARM_SSG/TARGET_MUSCA_S1/device/device_definition.c reason: Missing SPDX license identifier
File: /targets/TARGET_ARM_SSG/TARGET_MUSCA_S1/device/device_definition.h reason: Missing SPDX license identifier
File: /targets/TARGET_ARM_SSG/TARGET_MUSCA_S1/device/platform_base_address.h reason: Missing SPDX license identifier
File: /targets/TARGET_ARM_SSG/TARGET_MUSCA_S1/device/platform_description.h reason: Missing SPDX license identifier
File: /targets/TARGET_ARM_SSG/TARGET_MUSCA_S1/device/platform_irq.h reason: Missing SPDX license identifier
File: /targets/TARGET_ARM_SSG/TARGET_MUSCA_S1/device/platform_pins.h reason: Missing SPDX license identifier
File: /targets/TARGET_ARM_SSG/TARGET_MUSCA_S1/device/platform_regs.h reason: Missing SPDX license identifier
File: /targets/TARGET_ARM_SSG/TARGET_MUSCA_S1/device/system_core_init.c reason: Missing SPDX license identifier
File: /targets/TARGET_ARM_SSG/TARGET_MUSCA_S1/device/system_core_init.h reason: Missing SPDX license identifier
File: /targets/TARGET_ARM_SSG/TARGET_MUSCA_S1/device/drivers/cache_drv.c reason: Missing SPDX license identifier
File: /targets/TARGET_ARM_SSG/TARGET_MUSCA_S1/device/drivers/cache_drv.h reason: Missing SPDX license identifier
File: /targets/TARGET_ARM_SSG/TARGET_MUSCA_S1/device/drivers/gpio_cmsdk_drv.c reason: Missing SPDX license identifier
File: /targets/TARGET_ARM_SSG/TARGET_MUSCA_S1/device/drivers/gpio_cmsdk_drv.h reason: Missing SPDX license identifier
File: /targets/TARGET_ARM_SSG/TARGET_MUSCA_S1/device/drivers/i2c_ip6510_drv.c reason: Missing SPDX license identifier
File: /targets/TARGET_ARM_SSG/TARGET_MUSCA_S1/device/drivers/i2c_ip6510_drv.h reason: Missing SPDX license identifier
File: /targets/TARGET_ARM_SSG/TARGET_MUSCA_S1/device/drivers/musca_s1_scc_drv.c reason: Missing SPDX license identifier
File: /targets/TARGET_ARM_SSG/TARGET_MUSCA_S1/device/drivers/musca_s1_scc_drv.h reason: Missing SPDX license identifier
File: /targets/TARGET_ARM_SSG/TARGET_MUSCA_S1/device/drivers/qspi_ip6514e_drv.c reason: Missing SPDX license identifier
File: /targets/TARGET_ARM_SSG/TARGET_MUSCA_S1/device/drivers/qspi_ip6514e_drv.h reason: Missing SPDX license identifier
File: /targets/TARGET_ARM_SSG/TARGET_MUSCA_S1/device/drivers/timer_cmsdk_drv.c reason: Missing SPDX license identifier
File: /targets/TARGET_ARM_SSG/TARGET_MUSCA_S1/device/drivers/timer_cmsdk_drv.h reason: Missing SPDX license identifier
File: /targets/TARGET_ARM_SSG/TARGET_MUSCA_S1/device/drivers/timer_gp_drv.c reason: Missing SPDX license identifier
File: /targets/TARGET_ARM_SSG/TARGET_MUSCA_S1/device/drivers/timer_gp_drv.h reason: Missing SPDX license identifier
File: /targets/TARGET_ARM_SSG/TARGET_MUSCA_S1/device/drivers/uart_pl011_drv.c reason: Missing SPDX license identifier
File: /targets/TARGET_ARM_SSG/TARGET_MUSCA_S1/device/drivers/uart_pl011_drv.h reason: Missing SPDX license identifier
File: /targets/TARGET_ARM_SSG/TARGET_MUSCA_S1/Libraries/mt25ql_flash_lib.c reason: Missing SPDX license identifier
File: /targets/TARGET_ARM_SSG/TARGET_MUSCA_S1/Libraries/mt25ql_flash_lib.h reason: Missing SPDX license identifier
File: /targets/TARGET_ARM_SSG/TARGET_MUSCA_S1/partition/flash_layout.h reason: Missing SPDX license identifier
File: /targets/TARGET_ARM_SSG/TARGET_MUSCA_S1/partition/image_macros_preprocessed_ns.c reason: Missing license header
File: /targets/TARGET_ARM_SSG/TARGET_MUSCA_S1/partition/image_macros_preprocessed_s.c reason: Missing license header
File: /targets/TARGET_ARM_SSG/TARGET_MUSCA_S1/partition/region_defs.h reason: Missing SPDX license identifier
```

Offenders

Reply:
I'll improve Travis reports (new files without a proper license will report error to Travis), I'll create a new PR later today

Reply:
I have added the SPDX license identifiers to all the files on the list.